### DISCUSSION TITLE
allow only quality setting for image optimization

### DISCUSSION THREAD

**Discussion by juancresc** [SmileyChris/easy-thumbnails#595] [Ideas]
As the title says, I'd like an alias/option that allows me to just decrease the quality of the image as well as use optimize commands, but without changing its size. 

Right now if I don't specify a size option, I cannot generate thumbnails. 

this would be useful cause I'd be able to use this lib just for image optimization.

**Comment by SmileyChris** [comment]
I'd suggest just choosing a very large image size. It won't upscale images by default so you'll get the behaviour you want.
